lms/djangoapps/instructor/views/api.py

```diff
@@ -1092,6 +1092,42 @@ def re_validate_invoice(obj_invoice):
 @ensure_csrf_cookie
 @cache_control(no_cache=True, no_store=True, must_revalidate=True)
 @require_level('staff')
+def get_issued_certificates_features(request, course_id):  # pylint: disable=invalid-name
+    """
+    Responds with JSON if CSV is not required
```

Comment:
Nit: use the Google-style doc string comments:

https://github.com/edx/edx-platform/wiki/Python-Guidelines